packages/react-ui/internal/InternalMenu/InternalMenu.tsx

```diff
@@ -31,6 +31,8 @@ interface MenuProps {
   onItemClick?: (event: React.SyntheticEvent<HTMLElement>) => void;
   width?: number | string;
   preventWindowScroll?: boolean;
+  // Выключает добавление паддинга, пунктам меню без иконок, если у других пунктов меню есть иконка
+  preventIconPadding?: boolean;
```

Comment:
Предлагаю немного изменить нейминг здесь. Как показывает практика, пропы которые включают что-то проще для восприятия, чем пропы которые что-то выключают

```suggestion
  /**
   * Выравнивает текст всех пунктов меню относительно друг друга
   * Так, если хотя бы у одного пункта меню есть иконка, текст в  остальных пунктах меню будет выровнен относительно пункта меню с иконкой
   *
   * @default true
   */
  enableTextAlignment?: boolean;
```